frontend/packages/ux-editor/src/components/config/ConditionalRenderingComponent.tsx

```diff
@@ -254,29 +256,29 @@ class ConditionalRendering extends React.Component<IConditionalRenderingComponen
     const selectedMethod = this.state.conditionalRendering.selectedFunction;
     const selectedMethodNr = this.state.selectedFunctionNr;
     return (
-      <div className='modal-content'>
-        <div className='modal-header a-modal-header'>
-          <div className='a-iconText a-iconText-background a-iconText-large'>
-            <div className='a-iconText-icon'>
-              <i className='fa fa-corp a-icon' />
-            </div>
-            <h1 className='a-iconText-text mb-0'>
-              <span className='a-iconText-text-large'>
-                {this.props.language['ux_editor.modal_configure_conditional_rendering_header']}
-              </span>
-            </h1>
-          </div>
+      <Modal
+        isOpen={true}
+        onRequestClose={() => {}}
+        className={classes.modalBody}
+        ariaHideApp={false}
+        overlayClassName={classes.reactModalOverlay}
+      >
+        <div className={classes.modalHeader}>
+          <i className='fa fa-corp' />
+          <h1 className={classes.modalHeaderTitle}>
+            {this.props.language['ux_editor.modal_configure_conditional_rendering_header']}
```

Comment:
Could be exchanged with a `t()` function or similar. This goes for the whole file. But don't know how hard that is in a class component.

Reply:
I suggest we wait for that until we have converted this to a function component, it will be a lot easier then.

Reply:
@mijohansen, I absolutely agree with you, we should use the t() function. But as @TomasEng said it's easier to wait until this is a function component.